core/src/main/java/uk/ac/bbsrc/tgac/miso/core/service/printing/Driver.java

```diff
@@ -327,13 +327,19 @@ private static String getBarcode64(Barcodable barcodable) throws UnsupportedEnco
     return new String(Base64.encodeBase64(getBarcode(barcodable).getBytes("UTF-8")));
   }
 
-  private static void multiline(int lineLength, int maxLines, String input, BiConsumer<Integer, String> writer) {
-    for (int line = 0; line < maxLines - 1 && input.length() > line * lineLength; line++) {
+  private static int multiline(int lineLength, int maxLines, String input, BiConsumer<Integer, String> writer) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this method into the anonymous class declared at line 74. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3398)